### PR TITLE
Add authors to Project.toml

### PR DESCRIPTION
**Pull request by simonbyrne**
As discussed on Slack, I've added the current authors (sorted by contributions)

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 name = "ClimaComms"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-authors = ["CliMA Contributors <clima-software@caltech.edu>"]
+authors = [
+    "Kiran Pamnany <clima-software@caltech.edu>",
+    "Simon Byrne <simonbyrne@caltech.edu>",
+    "Charles Kawczynski <clima-software@caltech.edu>",
```
Comment (simonbyrne) [inline Project.toml:6]: @charleskawczynski what email would you like me to use?

File: Project.toml [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+    "Sriharsha Kandala <clima-software@caltech.edu>",
```
Comment (simonbyrne) [inline Project.toml:7]: @sriharshakandala what email would you like me to use?

File: Project.toml (hunk 1 above)
Comment (charleskawczynski) [reply Project.toml:6]: My Caltech one is fine: `charliek@caltech.edu`

Comment (simonbyrne) [reply Project.toml:6]:
```suggestion
    "Charles Kawczynski <charliek@caltech.edu>",
```

File: Project.toml (hunk 2 above)
Comment (sriharshakandala) [reply Project.toml:7]: Sriharsha.kvs@gmail.com . Thanks.

File: Project.toml [hunk 3]
```diff
@@ -1,6 +1,12 @@
 name = "ClimaComms"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-authors = ["CliMA Contributors <clima-software@caltech.edu>"]
+authors = [
+    "Kiran Pamnany <clima-software@caltech.edu>",
```
Comment (simonbyrne) [inline Project.toml:4]: @kpamnany would you like your personal email here (and on other repos)? or happy to leave the clima one?

File: Project.toml [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+    "Simon Byrne <simonbyrne@caltech.edu>",
+    "Charles Kawczynski <charliek@caltech.edu>",
+    "Sriharsha Kandala <clima-software@caltech.edu>",
+    "Jake Bolewski <clima-software@caltech.edu>",
```
Comment (simonbyrne) [inline Project.toml:8]: @jakebolewski I'm adding authors to the repo: would you like your personal email here (and on other repos)? or happy to leave the clima one?

File: Project.toml (hunk 2 above)
Comment (simonbyrne) [reply Project.toml:7]:
```suggestion
    "Sriharsha Kandala <Sriharsha.kvs@gmail.com>",
```

File: Project.toml (hunk 4 above)
Comment (simonbyrne) [reply Project.toml:8]: I'll merge it as-is for now. If you want to change it later, please let me know.

**Pull request merged**